kusd/validator/states.go

```diff
@@ -104,7 +119,7 @@ func (val *Validator) newElectionState() stateFn {
 	// @NOTE (rgeraldes) - wait for txs - sync genesis validators, round zero for the first block only.
 	if val.blockNumber.Cmp(big.NewInt(1)) == 0 {
 		numTxs, _ := val.backend.TxPool().Stats() //
-		if val.round == 0 && numTxs == 0 {        //!cs.needProofBlock(height)
+		if val.round == 0 && numTxs == 0 { //!cs.needProofBlock(height)
```

Comment:
if you change the aligned comment I think you can remove the one from the previous line 😃

Reply:
:+1: this was a refactor using my ide tooling, and fmted after

Reply:
actually, there should be NO commented code